api/src/main/java/ai/djl/util/Float16Utils.java

```diff
@@ -124,4 +125,27 @@ public static float halfToFloat(short half) {
         }
         return Float.intBitsToFloat((half & 0x8000) << 16 | (exp | mant) << 13);
     }
+
+    /**
+     * Converts a byte array into a float array.
+     *
+     * @param bytes the byte array.
+     * @param endian whether the data is stored in big-endian or little-endian order.
+     * @return a float array.
+     */
+    public static float[] fromByteArray(byte[] bytes, boolean endian) {
```

Comment:
```suggestion
    public static float[] fromByteArray(byte[] bytes, ByteOrder endian) {
```

Reply:
There actually 3 endian type:
BIG, LITTLE and NATIVE

Reply:
https://docs.oracle.com/javase/8/docs/api/java/nio/ByteBuffer.html#order-java.nio.ByteOrder-

bo - The new byte order, either BIG_ENDIAN or LITTLE_ENDIAN.

NATIVE is BIG_ENDIAN or LITTLE_ENDIAN

Reply:
I got whether it's big-endian or little-endian from AudioFormat, so there's no native involved.

Reply:
You can choose to only pass in BIG/LITTLE to this API in AudioFactory, but the API should be able to accept any endian. The source of the input to this API may not only come from AudioSystem

Reply:
OK

Reply:
Done